cwatch.php

```diff
@@ -355,6 +448,13 @@ private function pushUserAndLicenses($vars, array $service_licenses = null)
                     : 0;
             }
 
+            // Provision the license assigned to this package
+            if ($package->meta->cwatch_package_type == 'single_license'
```

Comment:
Looks like $package can be null, but you assume it's always an object which also contains a meta object that always has the cwatch_package_type and cwatch_license_type properties